src/features/user/Account/History.tsx

```diff
@@ -155,6 +220,16 @@ export default function History({
                   })}
               </div>
             </div>
+            <MuiButton
+              variant="contained"
+              onClick={!isLoading ? handleIssueReceipts : undefined}
+            >
+              {isLoading ? (
+                <CircularProgress color="inherit" size={20} />
+              ) : (
+                t('me:issueReceipts')
+              )}
+            </MuiButton>
```

Comment:
Add some space between the button and the filters above.

![Screenshot 2023-02-14 at 3 39 20 PM](https://user-images.githubusercontent.com/44917347/218705503-5fd40887-5c2a-42a5-a60e-f0ddc862ac0d.png)

Reply:
Fixed here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1677/commits/7bfacc3711ed6c5dd3d17cc935eb54ab88a64e4d